test/ui/widgets/bottom_bar_test.dart

```diff
@@ -16,16 +16,22 @@ import 'package:notredame/core/constants/router_paths.dart';
 // HELPERS
 import '../../helpers.dart';
 
+// MOCKS
+import '../../mock/services/networking_service_mock.dart';
+
 NavigationService _navigationService;
+NetworkingServiceMock networkingService;
 
 void main() {
   group('BottomBar - ', () {
     setUp(() {
       _navigationService = setupNavigationServiceMock();
+      networkingService = setupNetworkingServiceMock() as NetworkingServiceMock;
```

Comment:
Sometimes you cast the return of `setupNetworkingServiceMock` and sometimes not. Could you change it to unified it?
Also, could you remove the private of `_navigationService` please? :smile:

Reply:
Done! 😄